lib/Doctrine/ODM/PHPCR/Translation/TranslationStrategy/AttributeTranslationStrategy.php

```diff
@@ -176,7 +176,7 @@ public function getLocalesFor($document, NodeInterface $node, ClassMetadata $met
         $locales = array();
         foreach ($node->getProperties("*{$this->prefix}*") as $prop) {
             $matches = null;
-            if (preg_match('/' . $this->prefix . ':(..)-[^-]*/', $prop->getName(), $matches)) {
+            if (preg_match('/' . $this->prefix . ':([a-z]{2}_?[a-z]{0,2})-[^-]*/', $prop->getName(), $matches)) {
```

Comment:
Hm, so are we saying that we want to preserve the users choice of captalization? I would prefer to normalize it to something rather than have `de_at` and `de_AT` coexisting (would make searches potentially unreliable f.e.).

I would say better to normalize it to lowercase and then the user can normalize it another way if they want. For BC I guess we would have to normalize here and when we create the property.